hazelcast-jet-core/src/main/java/com/hazelcast/jet/pipeline/test/TestSources.java

```diff
@@ -161,4 +187,59 @@ void fillBuffer(TimestampedSourceBuffer<T> buf) throws Exception {
             }
         }
     }
+
+    /**
+     * Returns a {@link com.hazelcast.jet.pipeline.StreamSource} for {@code
+     * long} values. The value creation is distributed across the cluster and
+     * is done using {@link StreamSourceLong}. All {@link StreamSourceLong}
+     * instances are created identically, one per cluster member. The returned
+     * source is designed to be used for high-throughput performance testing.
+     *
+     * @param itemsPerSecond how many items should be emitted each second
+     * @param initialDelay initial delay before emitting values
+     *
+     * @return {@link com.hazelcast.jet.pipeline.StreamSource} with {@code long} values
+     *          created in a distributed fashion
+     *
+     * @since 4.3
+     *
+     */
+    @Nonnull
+    public static StreamSource<Long> streamSourceLong(long itemsPerSecond, long initialDelay) {
```

Comment:
could these methods be moved to the top? we shouldn't have interleaving methods/inner classes

Reply:
done in d476f97f6c1cb3bdd945b2b0da124869086022da